doc/reference/cmdline.rst

```diff
@@ -261,6 +347,8 @@ Developer features
     * ``(i)p(u)db``: start the chosen python debugger in the code when an
       unexpected error is raised before logging and returning the error.
 
+    * ``werkzeug``:
```

Comment:
missing explanations

Reply:
The explanation is missing in other places (eg command line help). Can you suggest something?

Reply:
AFAIS in the code ``wergzeug`` option is used to return exception info in the response.  Please let me know what to write there or to remove the option from docs.

Reply:
Maybe "Display the full traceback on the frontend page in case of exception"

Reply:
fixed